azure-configurationservice/tests/conftest.py

```diff
@@ -0,0 +1,113 @@
+import os
+import logging
+from msrestazure.azure_exceptions import CloudError
+from azure.configuration import AzureConfigurationClient
+from azure.configuration import ConfigurationSetting
+
+PAGE_SIZE = 100
+
+
+class AzConfigTestData:
+    def __init__(self):
+        self.key_uuid = None
+        self.label1 = None
+        self.label2 = None  # contains reserved chars *,\
+        self.label_uuid = None
+        self.label1_data = []
+        self.label2_data = []
+        self.no_label_data = []
+
+
+def _add_for_test(app_config_client, kv):
+    exist = bool(list(app_config_client.list_configuration_settings(keys=[kv.key], labels=[kv.label])))
+    if exist:
+        _delete_from_test(app_config_client, key=kv.key, label=kv.label)
+    return app_config_client.add_configuration_setting(kv)
+
+
+def _delete_from_test(app_config_client, key, label):
+    try:
+        app_config_client.delete_configuration_setting(key=key, label=label)
+    except CloudError:
+        logging.debug("Error occurred removing configuration setting %s %s" % (key, label))
+
+
+def _create_app_config_client():
+    connection_str = os.environ['AZ_CONFIG_CONNECTION']
+    return AzureConfigurationClient(connection_str)
+
+
+def setup_data():
+    test_data = AzConfigTestData()
+    app_config_client = _create_app_config_client()
+
+    key_uuid = "test_key_a6af8952-54a6-11e9-b600-2816a84d0309"
```

Comment:
Is this the data that we are using for live test?

Reply:
I use this pre-generated uuid for both live test replay. If the uuid is newly generated for every run, the replay can't find the matching request.